src/webextension/list/manage/components/account-summary.css

```diff
@@ -65,13 +65,13 @@
   /* XXX: This positioning is a hack and assumes that the <AccountSummary/> ends
      up in a particular spot in the UI. It would be nice if we could make this
      more flexible in the future. */
-  top: 50px;
-  right: 2px;
-  min-width: 130px;
-  padding: 8px 12px;
-  border: 1px solid #b1b1b3;
+  top: 60px;
+  right: 16px;
```

Comment:
Why these changes? I intentionally set the top/right position to match the visuals in #442, so that the caret beside the username lines up with the arrow in the "speech bubble" of the dropdown.

Reply:
the latest [mockups](https://zpl.io/25gqr00) remove the chevron and reposition the avatar, which then makes these "speech bubble" alignment changes more understandable.

It would be helpful if this PR included those, but they also go beyond simply CSS changes.  @nickbrandt would you like some help on that?